src/Views/scstormrunoff.html

```diff
@@ -0,0 +1,141 @@
+                                </div>
+                                <label class="fullwidth alignc orange" ng-if="!vm.validateForm(BohmanUrbanForm)"><span class="fa fa-exclamation-triangle"></span>&nbsp;&nbsp;Parameters Must Be Entered Before Results Can Be Calculated</label>
+                                <label ng-if="vm.validateForm(BohmanUrbanForm)">&nbsp;&nbsp;</label> 
+                            </form> 
+                            <div ng-if="vm.showResults">
+                                <div style="position:relative;">
+                                    <p class="fullwidth alignc hidden-print" style="font-size: 16px;"><b>Bohman Urban (1992) using {{vm.SelectedAEP.name}} AEP</b></p>
+                                    <div class="well">
+                                        <h3>Parameters</h3>
+                                        <table id="BohmanUrbanParameterTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Drainage Area (square miles):</td>
+                                                    <td><b>{{vm.drainageArea}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Main Channel Length (miles):</td>
+                                                    <td><b>{{vm.mainChannelLength}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Main Channel Slope (feet per mile):</td>
+                                                    <td><b>{{vm.mainChannelSlope}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Total impervious area (%):</td>
+                                                    <td><b>{{vm.totalImperviousArea}}</b></td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                        <h3>Summary</h3>
+                                        <table id="BohmanUrbanSummaryTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Weighted Runoff Volume:</td>
+                                                    <td><b>{{vm.ReportData.BohmanUrban1992.WeightedRunoff}}</b></td>
```

Comment:
Reduce the number of decimal places to 5 for the Weighted Runoff Volume (we may need to adjust this again later based on feedback from Kitty and cooperator)